src/Pane.tsx

```diff
@@ -46,8 +47,26 @@ export const InnerPane = ({
   bgColor,
   borderWidth,
   borderColor,
-  children
+  children: _children
 }: PropsWithChildren<InnerPaneProps>) => {
+  const children = useMemo(() => {
+    if (!_children) return _children;
```

Comment:
How about using `isValidElement`?

Something like this,
```ts
if (!isValidElement(_children))
  // Not a valid react element, so do something...
if (typeof _children.type === 'string')
  // Not a react component, so do something...
if ('displayName' in _children.type && _children.type.displayName === 'PaneSystem')
  // Found PaneSystem
```

We might need to check if the child PaneSystem is the only existing child in that Pane, otherwise the width would be wrong...

Reply:
Right! Modified to ensure PaneSystem is the sole child of Pane.

Also added context for warning when this rule isn't met.  
  
Decided not to check children directly (see https://react.dev/reference/react/Children). Went with context-based warnings instead.